src/logging/default-logger.ts

```diff
@@ -306,4 +306,8 @@ export class DefaultLogger implements Logger {
       }
     }
   }
+
+  printMetrics(): void {
```

Comment:
When this method is empty, eslint complains:

> Unexpected empty method 'printMetrics'.eslint@typescript-eslint/no-empty-function

This could also be resolved by adding `//eslint-disable-line` or updating the ESLint config, but adding a comment seemed pragmatic.

Reply:
+1